config/database.yml

```diff
@@ -19,21 +19,28 @@ test:
   host: localhost
   port: 5432
 
+deploy_default: &deploy_default
+  adapter: postgresql
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 }.to_i + 1 %>
+  timeout: 5000
+  url: <%= ENV["DATABASE_URL"] %>
```

Comment:
current `DATABASE_URL` looks like: `pya-production-web.cluster-<blah>.us-east-1.rds.amazonaws.com`
not like `adapter://user:password@host:port/database_name` which seems more classic, is this an issue?

@jamesiarmes @jnf ?

Reply:
> so here's my thoughts currently:
I think i currently have the url for the DATABASE_HOST not DATABASE_URL. this means that I need to remove DATABASE_URL from the ENV (this was set up in open tofu infra) and rename it to DATABASE_HOST so it doesn't use the wrong DATABASE_URL which will override everything in database.yml
Then I'll define DATABASE_HOST - using the url pya-production-web.cluster one (I already have DATABASE_URSER and DATABASE_PASSWORD ) in the deployed_defaults and remove DATABASE_URL completely. With them together I think we can connect to the correct database. 

chatted with Mike, proceeding with the above plan, currently making changes to opentofu infra: 
https://cfastaff.slack.com/archives/C0544ERAFQV/p1750788679445189?thread_ts=1750700709.865959&cid=C0544ERAFQV